styles/_editorjs.scss

```diff
@@ -107,6 +107,16 @@
     }
   }
 
+  .ce-inline-tool-input {
+    background-color: var(--lightBlack);
+    color: var(--mediumGray);
+    border: 000.1px solid #fff;
```

Comment:
it is not correctly styled, you are styling the wrapper and giving the same border. It is not OK